tests/components/sensor/test_islamic_prayer_times.py

```diff
@@ -0,0 +1,112 @@
+"""The tests for the Islamic prayer times sensor platform."""
+from homeassistant.util.async_ import run_coroutine_threadsafe
+from homeassistant.setup import setup_component
+from tests.common import get_test_home_assistant
+from homeassistant.components.sensor.islamic_prayer_times import (
+    IslamicPrayerTimesData, IslamicPrayerTimeSensor)
+from tests.common import MockDependency
+
+LATITUDE = 41
+LONGITUDE = -87
+CALC_METHOD = 'isna'
+PRAYER_TIMES = {"Fajr": "06:10", "Sunrise": "07:25", "Dhuhr": "12:30",
+                "Asr": "15:32", "Maghrib": "17:35", "Isha": "18:53",
+                "Midnight": "00:45"}
+
+
+class TestIslamicPrayerTimesSensor():
```

Comment:
We prefer standalone pytest test functions for new tests. See the deconz test suite for example.